samples/nginx.conf

```
@@ -0,0 +1,13 @@
+server {
+    listen 80;
```

```suggestion
    listen 80;
    listen 81;
```

safe to listen to a range? or does nginx provide way to roll over to next port automatically if current is in use?

I am suggesting this way because I feel all containers could communicate in host network mode, instead of specifying port mapping between containers. This is much more efficient via kubernetes deployment

---

This sample nginx is running inside container, we have full control on what port to use.